pom.xml

```diff
@@ -31,7 +31,7 @@
         <vertx.auth.jwt.version>4.0.3</vertx.auth.jwt.version>
 
         <!-- Micronaut minimal app -->
-        <micronaut.version>2.4.2</micronaut.version>
+        <micronaut.version>2.3.0</micronaut.version>
```

Comment:
Why reduce the micronaut version? Is that intentional?

Reply:
@jerboaa I wanted to find a version that actually works without warnings with some known GraalVM version. 2.4.2 is not that. So I reverted back to what I knew as a working version. Lemme address it later in https://github.com/Karm/mandrel-integration-tests/issues/39

Reply:
OK.